tests/Test.MSAL.NET.Unit/PublicClientApplicationTests.cs

```diff
@@ -709,5 +709,27 @@ public void AcquireTokenSilentServiceErrorTest()
                 Assert.IsTrue(HttpMessageHandlerFactory.IsMocksQueueEmpty, "All mocks should have been consumed");
             }
         }
+
+        [TestMethod]
+        [TestCategory("PublicClientApplicationTests")]
+        public void GetUserTest()
+        {
+            var app = new PublicClientApplication(TestConstants.ClientId);
+            var users = app.Users;
+            Assert.IsNotNull(users);
+            Assert.AreEqual(0, users.Count());
+
+            TokenCacheHelper.PopulateCache(app.UserTokenCache.TokenCacheAccessor);
+            users = app.Users;
+            Assert.IsNotNull(users);
+            Assert.AreEqual(1, users.Count());
```

Comment:
you should add 2 users to the cache to make sure that it is truly returning the correct user.